src/api/routes/company.js

```diff
@@ -72,6 +72,15 @@ export default (app) => {
 
     });
 
+    router.get("/:companyId/profile", validators.profile, async (req, res) => {
+        const company = await new CompanyService().findById(req.params.companyId, req.hasAdminPrivileges, req.hasAdminPrivileges);
+        let offers = await new OfferService().getOffersByCompanyId(req.params.companyId, req.params.companyId, req.hasAdminPrivileges);
```

Comment:
```suggestion
        let offers = await new OfferService().getOffersByCompanyId(req.params.companyId, req.params.companyId, req.hasAdminPrivileges).current();
```
I think this is enough to only show current offers but please test it. This approach means that not even the owner sees unpublished offers on his profile but I think that makes sense, since we already have a `My Offers` page

Reply:
I believe this is related to what I pointed out in another review comment, but I was unable to do this since `getOffersByCompanyId` is currently not returning the query, hence `current()` will not be a usable function.